apps/test/unit/templates/sectionProgress/progressTables/ProgressTableViewTest.jsx

```diff
@@ -177,6 +179,13 @@ describe('ProgressTableView', () => {
       expect(wrapper.find(ProgressLegend)).to.have.length(1);
     });
 
+    it('renders review state badges in ProgressLegend when enabled', () => {
```

Comment:
I'd actually prefer to check that the right prop is passed to `ProgressLegend` here rather than testing what `ProgressLegend` renders (since the ProgressLegend tests already do that)

Reply:
👍